1.0.1 Changes

cs-abhishek-shukla opened the pull request:
#### Description
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots
- Added release notes
- Edited `.gitignore` to remove `.idea/` from the last line (`.idea/.DS_Store`)

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,27 +1,28 @@
+| Icon                           | Rich-text field containing the icon image.                                                                                                                                                                                                                                                                                                                                   ...
+| Title                          | Field containing the title of the card. The card displays the value set as the title for that record.                                                                                                                                                                                                                                                                        ...
+| Subtitle                       | Field containing a short description of the card. The card displays the value set as the description of that record.                                                                                                                                                                                                                                                         ...
+| Card Left Border               | Field containing the left border color code of the card. |
```
Please change it "Left border" only.

cs-abhishek-shukla replied:
Changed

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -1,9 +1,9 @@
 # Usage
 
-The Record Card widget helps visualize records based on a specified data source. An example of using the record widget would be displaying a list of tasks in the Record Card view on the Production and Development tab of the Continuous Delivery page.
+The **Record Card** widget helps visualize records based on a specified data source. An example of using the record widget would be displaying a list of tasks in the **Record Card** view on the Production and Development tab of the Continuous Delivery page.
```
Can we add this as an example you can use it with continuous delivery.

It looks like this widget is only meant for continuous delivery. Can we discuss this ?

cs-abhishek-shukla replied:
Done.

cs-abhishek-shukla committed:
1.0.1 Changes
    - Added full screen images
    - Rewrote setup, readme, and Usage
    - Annotated screenshots
    - Added release notes

cs-suyog-jinturkar reviewed (approved):
Dev approved it
